### PR TITLE
Safemode - Fix indicator when switching units

### LINKED ISSUE #3807: Safe mode indicator broken when remote controlling a unit
**Arma 3 Version:** `1.58` stable
**CBA Version:** `2.3.1` stable
**ACE3 Version:** `3.5.1` stable

**Mods:**
- `@CBA_A3`
- `@ace`

**Description:**
When remote controlling a unit after putting your weapon on safe the firing mode indicator will stay empty, even though the weapon is not on safe.

**Steps to reproduce:**
- Be Zeus
- Put your weapon on safe
- Remote control a unit
- Press F, firing mode indicator stays empty even though you can fire your weapon
- Press keybind to put your weapon on safe _twice_
- Firing mode indicator reappears

**Where did the issue occur?**
- Dedicated

**Placed Modules:**
- On demand

**RPT log file:**
- On demand

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
**When merged this pull request will:**
- Fix #3807

### CODE REVIEW

**Comment by BaerMitUmlaut** [comment]
Oh also I removed the todo because that seems to be a leftover from AGM about this exact problem.

File: addons/safemode/XEH_postInit.sqf [hunk 1]
```diff
@@ -18,6 +12,9 @@ if (!hasInterface) exitWith {};
     // Statement
     [ACE_player, currentWeapon ACE_player, currentMuzzle ACE_player] call FUNC(lockSafety);
     true
-},
-{false},
-[41, [false, true, false]], false] call CBA_fnc_addKeybind;
+}, {false}, [41, [false, true, false]], false] call CBA_fnc_addKeybind;
```
Comment (Dystopian) [inline addons/safemode/XEH_postInit.sqf]: Magic variable, should be something like `DIK_*`.

**Comment by BaerMitUmlaut** [comment]
Okay, so when would that actually happen? I can't find it ever being raised with "Soldier".

**Comment by commy2** [comment]
>Okay, so when would that actually happen?
Loading a savegame for example.

**Pull request merged**